src/tclint/cli/tclsp.py

```diff
@@ -205,6 +205,7 @@ def format(
         formatter = Formatter(
             FormatterOpts(
                 indent=indent,
+                indent_mixed_tab_size=0,
```

Comment:
I think we'll eventually want `config.get_indent_mixed_tab_size()` here, although I'm happy to add this in a follow-up PR after I have a chance test it (per #127, I understand you don't have a convenient way to test this right now).